day_5/Day_5_Highest_Score.py

```diff
@@ -0,0 +1,12 @@
+list_of_numbers = [1,2,3,4,5,6,7]
```

Comment:
Well, well, well. Lets change this task a little. You already know how to generate a random numbers so my task for you is to create a list of random integer numbers from `1` to `1000` of size `500`. Then the following(same) logic will be used  on a new list with random numbers.